packages/core/src/metrics/platform/CordovaAppMetrics.ts

```diff
@@ -1,15 +1,16 @@
 import { AppMetrics, Metrics } from "../model";
 
+// plugin's JS interface
+declare var aerogear: {
+  getAppMetrics(): Promise<AppMetrics>;
```

Comment:
Can this be defined in plugin types rather than here.

Reply:
You're right, this was more a workaround than anything else, anyway -> https://github.com/aerogear/aerogear-js-sdk/pull/66#discussion_r184704632